dScripts/BuccaneerValiantShip.cpp

```diff
@@ -0,0 +1,24 @@
+#include "BuccaneerValiantShip.h"
+#include "SkillComponent.h"
+#include "dLogger.h"
+
+void BuccaneerValiantShip::OnStartup(Entity* self) {
+    const auto skill = 982;
+    const auto behavior = 20577;
+    const auto skillCastTimer = 1.0F;
+
+    self->AddCallbackTimer(skillCastTimer, [self]() {
```

Comment:
Have to capture skill and behavior here if you want to use them within the lambda. The current implementation isn’t standard.